lib/data-types.js

```diff
@@ -345,11 +345,13 @@ inherits(DECIMAL, NUMBER);
 
 DECIMAL.prototype.key = DECIMAL.key = 'DECIMAL';
 DECIMAL.prototype.toSql = function toSql() {
+  let colDefinition = 'DECIMAL';
```

Comment:
Like this https://github.com/sequelize/sequelize/blob/master/lib/data-types.js#L187